google/ggrc-core
# GGRC-5564 Hotlist ID isn't synced to a ticket

Replace 'hotlist_id' with 'hotlist_ids' in query to issue tracker.



# Issue description

Hotlist ID isn't synced to a ticket

# Steps to test the changes

1. Create a GGRC Issue object with:

hotlist id: 700706
component id: 64445
Make sure that you have enough rights for hotlist to add it on tickets.

2. Go to ticket on clicking 'Ticket' link on Issue info pane.

Actual result: Ticket isn't added to a specified hotlist.

Expected result: Ticket should be added to a specified hotlist.

3. Go to a GGRC Issue object from step 1.

4. Open edit pop-up and update a hotlist to a new one '714882'.

5. Save changes.

Actual result: Error message 'Unable to update a ticket in issue tracker.' is shown. Ticket isn't added to a specified hotlist.

Expected result: No error messages should be shown. Ticket should be added to a specified hotlist

# Solution description

Fix typo query sent to issue tracker.

# Sanity checklist




# PR Review checklist
